misp_modules/modules/import_mod/__init__.py

```diff
@@ -15,4 +15,5 @@
     'threatanalyzer_import',
     'csvimport',
     'joe_import',
+    'trustar_import',
```

Comment:
Is there any trustar import module you want to submit as well?

Reply:
Apologies, this module started as an import until we realized we would like it to enrich data rather than just import it. There is no import module planned at this time.